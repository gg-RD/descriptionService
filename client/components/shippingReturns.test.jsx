import React from 'react';
import { shallow, mount, render } from 'enzyme';
import ShippingReturns from './shippingReturns.jsx'


describe('ShippingReturns Component', () => {
	it('ShippingReturns Component should render as expected', () => {
		const wrapper = shallow(<ShippingReturns />)
		expect(wrapper).toMatchSnapshot();
	});

});ShippingReturns