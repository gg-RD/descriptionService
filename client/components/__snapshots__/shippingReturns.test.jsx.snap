// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingReturns Component ShippingReturns Component should render as expected 1`] = `
<div>
  Free standard shipping and 30-day free returns, only with NikePlus. Exclusions apply. Learn more.
  <li>
    Standard / Arrives 2-4 Business Days 
  </li>
  <li>
    Two-Day / Arrives 2-3 Business Days 
  </li>
  <li>
    Next-Day / Arrives 1-2 Business Days 
  </li>
</div>
`;
